src/pages/Home/index.js

```diff
@@ -0,0 +1,35 @@
+import { useNavigate } from "react-router-dom";
+
+const Home = () => {
+
+    const navigate = useNavigate();
+
+    const joinOnline = () => {
+        navigate('/online', { replace: true });
+    }
+
+    const joinOffline = () => {
+        navigate('/offline', { replace: true });
+    }
+
+    return (
+        <div>
```

Comment:
add `marginTop`, I think better to place these 2 buttons in the center of the screen